oviewer/convert_es.go

```diff
@@ -90,6 +96,9 @@ func (es *escapeSequence) convert(st *parseState) bool {
 		case mainc >= '0' && mainc <= 'f':
 			es.parameter.WriteRune(mainc)
 			return true
+		case mainc < 0x40:
```

Comment:
This is a magical number here, you should either explain why it's 0x40, or add a constant with a meaningful name

0x40 is @ right? 0x41 is A

What is this if statement about?

Checking that anything before @ is handled in a common way? But then @ is not caught?

I'm still unsure about what the code does